### PR TITLE
Samsung.play av station 4.1.20.47 2ec73e50 ce84 4345 8c2f 7fb49855d65d

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113300&drop=dogfoodAlpha

### CODE REVIEW

**Comment by ItzLevvie** [comment]
[Policy] Needs-Author-Feedback

**Comment by upintheairsheep** [comment]
> 

Hardware dependency

**Comment by stephengillie** [comment]
> Manifest Error: Schema Error. [Schema validation failed. 
> Error context: <root>[Installers][0][InstallerSha256] 
> Description: Value type not permitted by 'type' constraint. 
> Error context: <root>[Installers][0] 
> Description: Failed to validate against schema associated with property name 'InstallerSha256'. 
> Error context: <root>[Installers] Description: Failed to validate item #0 in array. 
> Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: Sa***ung.PlayAVStation.installer.yaml 
> Manifest Error: Required field missing. [InstallerSha256]

File: manifests/s/Samsung/PlayAVStation/4.1.20.47/Samsung.PlayAVStation.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: Samsung.PlayAVStation
+PackageVersion: 4.1.20.47
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=1908722&CDCttType=SW&ModelType=C&ModelName=NP-Q1U/P01/SEA&VPath=SW/200804/20080422161039281_Play_AVStation_4.1.20.47.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: msi
+  NestedInstallerFiles:
+  - RelativeFilePath: Play AVStation.msi
+  InstallerSha256:
```
Comment (stephengillie) [inline manifests/s/Samsung/PlayAVStation/4.1.20.47/Samsung.PlayAVStation.installer.yaml:13]:
```suggestion
  InstallerSha256: 4B873EC9C00ED973679FB2244A2EB91E23263A53E7802A2BFABA292F577B548D
```

Comment (stephengillie) [reply manifests/s/Samsung/PlayAVStation/4.1.20.47/Samsung.PlayAVStation.installer.yaml:13]: This got snipped in the last commit.

**Comment by Trenly** [comment]
> > 
> 
> Hardware dependency

[Policy] Hardware

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by denelon** [comment]
Supporting custom hardware in validation is "Not Planned".
* https://github.com/microsoft/winget-pkgs/issues/199504

**Pull request closed without merging**